main.go

```diff
@@ -76,7 +76,14 @@ func registerSIGINTHandler(mountPoint string) {
 }
 
 func getUserAgent(appName string) string {
-	return strings.TrimSpace(fmt.Sprintf("gcsfuse/%s %s %s", getVersion(), appName, os.Getenv("GCSFUSE_METADATA_IMAGE_TYPE")))
+	if len(os.Getenv("GCSFUSE_METADATA_IMAGE_TYPE")) > 0 {
```

Comment:
We are calling os.Getenv("GCSFUSE_METADATA_IMAGE_TYPE") twice. So can we create variable and use it at both places ?

Reply:
Done, thanks!